CONTRIBUTING.rst

```diff
@@ -82,3 +82,19 @@ coverage rate should increase or stay the same. Automatic services will ensure
 your code works on all the operative systems and package combinations
 poliastro support - specifically, note that poliastro is a Python 3 only
 library.
+
+Development environment
+-----------------------
+
+To set up a development environment:
+
+1. `Install git <https://git-scm.com/>`_ on your computer.
+2. `Register to GitHub <https://github.com/>`_.
+3. `Fork poliastro <https://help.github.com/articles/fork-a-repo/>`_.
+4. `Clone your fork <https://help.github.com/articles/cloning-a-repository/>`_.
+5. Install it in development mode using :code:`pip install --editable .` (this
```

Comment:
Maybe I would change `pip install --editable .` to `pip install --editable /path/to/cloned/repo` or something like that. Actually I'm being a little nitpicking (it's my first review, I have to write something), I really like all of the changes 😄